### ISSUE TITLE
[Bug]: Cartography loading map crashes game

### ISSUE THREAD

**Issue by Kappadar** [MelvorIdle/melvoridle.github.io#3819]
### Describe the bug

All mods disabled, level 102 cartography. Every time I try to load it (pressing on cartography skill) the map loads to an arbitrary #% (But doesn't finish loading the map, just stops at like 67% for example) then the entire game freezes and I can't press anything else. I can't even close out window, have to stop the process in task manager. 

### Reproduction Steps

1. Open Melvor Idle
2. Click Cartography skill (I am level 102, didn't have this issue at lower levels.)
3. Map will display "loading x%"
4. Game will freeze, only loading gif will move, can't click on anything else

### Expected behaviour

1. Open Melvor idle
2. Click Cartography skill
3. Map will display "loading x%"
4. Map will finish loading, then proceed to play as intended

### Save Link - Required if issue occurs with a save

https://melvoridle.com/save/idsM863F

### Screenshots

![image](https://github.com/MelvorIdle/melvoridle.github.io/assets/51834268/83d3e2a8-dc72-4b1b-a696-a3d22e96ff83)
![image](https://github.com/MelvorIdle/melvoridle.github.io/assets/51834268/e2bc5b9a-b1e6-42c3-be48-63d5308aabc2)

Can only scroll through skills + screenshot.

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [X] Steam
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [ ] Mobile App (Android Beta)
- [ ] Other (Please Specify)

### Which expansions are purchased and active at the time of the bug?

- [X] Throne of the Herald
- [X] Atlas of Discovery

### Which version of the game are you experiencing this bug on?

v1.2.1

### Game Subversion

(?7994)

### Are you using any mods?

No

**Comment by sillyjoeblack** [comment]
Having the same issue. 
![image](https://github.com/MelvorIdle/melvoridle.github.io/assets/151890622/dbfa4daa-649a-40e0-9539-53d4656a6630)
I have relaunched numerous times, I have verified the game files in steam, I have restarted my computer, I have reinstalled Melvor entirely. Always the same issue, starts loading back into cartography and immediately crashes. Steam, Windows 10, both DLC's, no mods.

**Comment by Sargoth9** [comment]
I have exactly the same issue.

Started happening today (1.2.2024)
Game version: V1.2.2 (?8381)
Platform: Epic Games

I am playing on a good PC build, I even lowered Carto map to low details in options. I tried waiting for 5+ minutes for it to load, refreshing the game (means killing it in Task Manager and restarting) 5+ times. Nothing works.
I never had issues with Carto map so far, but it started happening this morning when I reached Carto lvl 102 overnight. I might have possibly completed the map I assume.

EDIT - Just before I was about to post this I tried maybe 10 more times closing/reloading the game and it finally loaded once for now. Still, very annoying bug.
